### ISSUE TITLE
FE: Accessibility and SEO: Dev console issues should be addressed for best practices.

### ISSUE THREAD

**Issue by bryson-palmer** [bryson-palmer/simple-todo#28]
When running the app in production and the dev console is open in chrome, the issues tab has a number of issues that need to be addressed. It would seem adding the appropriate attributes to elements would be the fix. Maybe there is a disconnect with the top level formik element not passing the name/id down properly. This is happening on the note form title & body inputs. For reference, it seems that the folder name input doesn't have any issues related to it's input.
